modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -54,243 +47,294 @@ namespace {
         std::optional<bool> maskingEnabled;
-
         // [[codegen::verbatim(MaskingQuantityInfo.description)]]
         std::optional<int> maskingQuantity;
-
-        // List of ranges for which their corresponding parameters values will be
-        // masked by. Should be entered as {min value, max value} per range
+        // Ranges for which their corresponding quantity parameter value will be
+        // masked by. Should be entered as {min value, max value}
         std::optional<std::vector<glm::vec2>> maskingRanges;
+        // Ranges for which their corresponding parameters values will be
+        // masked by. Should be entered as min value, max value
+        std::optional<glm::vec2> maskingMinMaxRange;
 
         // [[codegen::verbatim(DomainEnabledInfo.description)]]
         std::optional<bool> domainEnabled;
 
-        // Value should be path to folder where states are saved. Specifying this
-        // makes it use file type converter
-        // (JSON/CDF input => osfls output & oslfs input => JSON output)
-        std::optional<std::string> outputFolder;
-
         // [[codegen::verbatim(LineWidthInfo.description)]]
         std::optional<float> lineWidth;
 
+        // [[codegen::verbatim(ColorUseABlendingInfo.description)]]
+        std::optional<bool> alphaBlendingEnabled;
+
+        // Set if first/last file should render forever
+        bool showAtAllTimes;
+
+        // If using dynamic downloading, files are not cached by default.
+        // To save files locally , set this to true.
+        std::optional<bool> CacheData;
+
         // If data sets parameter start_time differ from start of run,
         // elapsed_time_in_seconds might be in relation to start of run.
         // ManuelTimeOffset will be added to trigger time.
-        std::optional<double> manualTimeOffset;
+        std::optional<float> manualTimeOffset;
+
+        enum class [[codegen::map(openspace::fls::Model)]] Model {
+            Batsrus,
+            Enlil,
+            Pfss
+        };
+        // Currently supports: batsrus, enlil & pfss. Not specified -> model == invalid
+        // which just means that the scaleFactor (scaleToMeters) will be 1.f assuming
+        // meter as input
+        std::optional<Model> simulationModel;
+
+        // Convert the models distance unit, ex. AU to meters for Enlil.
+        // 1.f is default, assuming meters as input.
+        // Does not need to be specified if simulationModel is specified.
+        // Using a different model? Set this to scale your vertex positions to meters.
+        std::optional<float> scaleToMeters;
+
+        // choose type of loading:
+        //0: static loading and static downloading
+        //1: dynamic loading and dynamic downloading
+        enum class [[codegen::map(openspace::RenderableFieldlinesSequence::LoadingType)]]
+        LoadingType {
+            StaticLoading,
+            DynamicDownloading
+        };
+        std::optional<LoadingType> loadingType;
```

Comment:
```suggestion
        // Decides whether to use static or dynamic data downloading.
        std::optional<LoadingType> loadingType;
```
Should also include details of what the different options mean